bin/morgue.js

```diff
@@ -834,10 +834,25 @@ function coronerLimit(argv, config) {
 }
 
 function tenantURL(config, tn) {
-  var ix = config.endpoint.indexOf('.');
-  var s = [config.endpoint.substr(0, ix), config.endpoint.substr(ix)];
+  /*
+   * Get tenant separator. In Coronerd <1.46, this is always "." because
+   * it wasn't exposed via /api/config.
+   */
+  var tsep = config.config.tenant_separator;
+  if (tsep === undefined)
```

Comment:
this won't handle all specific cases - like `null`. Common practice is to use object comparision
example:
```
if(tsep){
// it means that object exist
}
```
```
it(!tsep) {
// object doesn't exist
}
```